test/js/scp-api.test.js

```diff
@@ -1,17 +1,20 @@
-/* eslint-disable no-undef */
+/* eslint-disable*/
```

Comment:
Could you add a code comment briefly explaining why the linter is completely disabled in this file?

I suspect this is needed to avoid VS Code from buggily auto-inserting `Promise`, as we saw while pair-programming today.  But that should be noted in code, to avoid maintainers reflexively re-adding the linter.

If that buggy auto-inserting happens in more than this one file, then we should open an SCP ticket to find a better solution than disabling the linter.